addon/adapters/github.js

```diff
@@ -19,7 +19,8 @@ export default RESTAdapter.extend({
   }),
 
   pathForType(type) {
-    return Ember.String.camelize(Ember.String.pluralize(type.replace('github', '')));
+    return Ember.String.camelize(Ember.String.pluralize(type.replace('github', '')))
+      .replace('organizations', 'orgs');
```

Comment:
Maybe I should just remove the `replace` and add this to the `organizations` adapter.

```
pathForType() {
  return 'orgs';
}
```